app/v2/notifications/notification_schemas.py

```diff
@@ -122,7 +122,29 @@
     "definitions": {
         "notification": get_notification_response
     },
+}
 
+send_a_file_validation = {
+    "type": "object",
+    "additionalProperties": False,
+    "properties": {
+        "file": {
+            "type": "string",
+            "required": True,
+        },
+        "is_csv": {
+            "type": ["boolean", "null"],
+            "required": False,
+        },
+        "confirm_email_before_download": {
+            "format": "send_a_file_confirm_email",
+            "required": False,
+        },
+        "retention_period": {
+            "format": "send_a_file_retention_period",
```

Comment:
```suggestion
            "format": "send_a_file_retention_period",
            "type": ["string", "null"],
```

this could be a useful step to nudge users towards a string if they enter a number

Reply:
As above.